app/models/concerns/episode_media.rb

```diff
@@ -20,7 +99,7 @@ def cut_media_version!
     latest_ids = latest_media.map(&:id)
 
     # backfill media_versions for newly completed media
-    if media_ready? && latest_ids != media_ids
+    if media_ready?(true) && latest_ids != media_ids
```

Comment:
Okay, and here's the possibly-a-bug where it wasn't making sure the latest content was actually completed.